src/IB/IBFEMethod.cpp

```diff
@@ -1017,6 +1022,41 @@ IBFEMethod::spreadForce(const int f_data_idx,
         d_ghost_data_accumulator->accumulateGhostData(f_scratch_data_idx);
     }
 
+    // Prolong forces spread onto coarser levels onto finer levels.
+    if (d_use_scratch_hierarchy)
+    {
+        // not yet implemented
+        assertStructureOnFinestLevel();
+    }
+    else
+    {
+        // TODO - we should save these schedules in the same manner that we save
+        // the schedules for the scratch hierarchy.
+        for (int ln = getCoarsestPatchLevelNumber(); ln < getFinestPatchLevelNumber(); ++ln)
+        {
+            Pointer<RefineAlgorithm<NDIM> > refine_algorithm = new RefineAlgorithm<NDIM>();
+            Pointer<CartesianGridGeometry<NDIM> > geometry = d_hierarchy->getGridGeometry();
+            // TODO: this is probably the wrong refinement operator
+            Pointer<RefineOperator<NDIM> > refine_op =
+                geometry->lookupRefineOperator(f_var, "CONSERVATIVE_LINEAR_REFINE");
+            // TODO: do we need another scratch data index? We should be able to
+            // prolong directly into the output index
+            refine_algorithm->registerRefine(
+                f_prolong_scratch_data_idx, f_scratch_data_idx, f_prolong_scratch_data_idx, refine_op);
```

Comment:
```suggestion
                f_prolong_scratch_data_idx, f_prolong_scratch_data_idx, f_prolong_scratch_data_idx, refine_op);
```

Reply:
I don't understand this change. The second argument should be the source index - that's definitely `f_scratch_data_idx`. What am I missing?

Reply:
I misunderstood what is going on here --- I think that the current code is right.